components/FrontPage/Section/Cryosphere.vue

```diff
@@ -0,0 +1,21 @@
+<script lang="ts" setup></script>
+
+<template>
+  <FrontPageSectionWrapper tag="Cryosphere">
+    <div class="columns">
+      <div class="column border-right">
+        <ItemTextPicture slug="landfast-sea-ice" />
+      </div>
+      <div class="column">
+        <ItemTextPicture slug="sea-ice-concentration" />
+      </div>
+      <div class="column border-left">
+        <ItemTextPicture slug="notebook-sea-ice-traffic" />
+      </div>
+    </div>
```

Comment:
I noticed that this is the only FrontPage Section component that uses columns instead of tiles. I'm not sure if this was deliberate or not since this is also the only FrontPage Section with only three items. I think it might be better to use tiles anyway for consistency between FrontPage Sections (+ to help keep CSS styling and responsiveness consistent).

Reply:
Fixed!